### DISCUSSION TITLE
Отключение Openvpn

### DISCUSSION THREAD

**Discussion by shilophrenic** [xtrime-ru/antizapret-vpn-docker#74] [Q&A]
Подскажите, можно ли как-то правильно убрать openvpn контейнер и все конфиги связанные с ним, чтобы это никак не повлияло на работоспособность WG и AdGuard ? 
Полностью переключился на WG и не хочу иметь лишние открытые порты и запущенные контейнеры

**Comment by xtrime-ru** [comment]
Сейчас openvpn интегрирован в контейнер antizapret. Есть несколько вариантов: 

1. убрать проброс портов в docker-compose для контейнера antizapret-vpn
2. Удалить сборку и установку из Dockercompose и пересобрать образ
3. Выключить openvpn через systemctl

Возможно в будущем найдется время вынести openvpn в отдельный контейнер, тогда можно будет просто не запускать его.

**Reply by Ernieleo** [reply]
Можете подсказать вариант отключения OpenVPN на уже установленном контейнере, чтобы при перезапуске, в случае рестарта системы, OpenVPN не запускался вновь? Может, вам временно добавить в виде обновления параметр для его отключения?

Понятно, что OpenVPN идёт в базовой установке Антизапрета, но на данный момент, когда OpenVPN активно определяется и блокируется со стороны провайдеров, и при увеличении популярности VPN и борьбы с ним, было бы логично сделать основным решением рабочий вариант на Амнезии либо позволить конфигурировать установку, выбирая конкретный вариант. Антизапрет удобен тем что не нужно тратить время на составление списков туннелирования на каждом устройстве и не только, а в связке с рабочим протоколом VPN — это просто 10 из 10 решений.

**Reply by xtrime-ru** [reply]
1. убрать проброс портов в docker-compose для контейнера antizapret-vpn 

Например так: 
```yml
     ports: !reset []
```

Используйте файл docker-compose.override.yml что бы ваши правки не конфликтовали с будущими обновлениями git.

PS. Работа над выносом openvpn в отдельный контейнер ведется.
